### ISSUE TITLE
No Device Found

### ISSUE THREAD

**Issue by Tbails** [360Controller/360Controller#649]
Hi,
Really appreciate the effort gone in to put this together.

Unfortunately, I cannot get the controller to work on my MacBook pro.

Every time I try and connect it the controller vibrates for a second but no light comes on. 

I have read the HELPME file and still no luck. Tried using multiple cables and controllers as well. 

Any help would be appreciated.

**Comment by FranticRain** [comment]
Before we can help you, please consult the README and provide all necessary information so that we can help you with your issue. https://github.com/360Controller/360Controller#xbox-one-controllers-connected-with-usb

**Comment by Tbails** [comment]
Hi, i even ready the READMR However I am unclear as to exactly what it is you need in terms of info?

**Comment by FranticRain** [comment]
I need you to provide the "Product ID" and "Vendor ID" of the controller. These can be found by accessing the Apple menu, selecting "About this Mac", and then selecting "System Report..." on the "Overview" tab. On the left hand side of the new window, select the "USB" option under "Hardware". If the controller is plugged in, there should be an entry in this window called "Controller".

**Comment by Tbails** [comment]
It does not appear, when plugged in, on this screen. The xbox logo does not light up on the controller either it only vibrates for a moment.

**Comment by FranticRain** [comment]
> If it isn't listed there, then you likely have a "charge" Micro USB cable instead of a "data" cable. If the cable isn't sending data, then you can't use the driver. Try a different cable.
From the README.

**Comment by Tbails** [comment]
Hi again,

I now have a cable which works and have the following information
Product ID:	0x02d1
  Vendor ID:	0x045e  (Microsoft Corporation)

**Comment by FranticRain** [comment]
Please ensure that the driver has been allowed in System Preferences -> Security and Privacy -> General. This approval UI is only present in the Security & Privacy preferences pane for 30 minutes after the alert. Until you approve the kext, future load attempts will cause the approval UI to reappear but will not trigger another user alert.

![Security and Privacy panel in the macOS System Preferences](https://user-images.githubusercontent.com/33877645/33092253-965b666e-cec7-11e7-940d-3e13b46ca500.png)

**Comment by Tbails** [comment]
Right, but I have not had this software UI, is there any way to trigger it or check if I have allowed it before?

**Comment by FranticRain** [comment]
I'm not sure there's any way to check if you have allowed it already, but the way to prompt this UI is to connect the controller after the driver has been installed. This UI will only show up in 10.13.4 and later, I think. I forget when exactly it was implemented. 

I guess the best way to check if you have approved it is to run `sudo kextstat | grep com.mice ` in your terminal fairly soon after connecting the controller. If the result is empty, then the driver hasn't loaded. If you see an entry, then the driver has been properly loaded.

**Comment by Tbails** [comment]
Does not look like there is anything 
Last login: Mon May 14 18:38:46 on ttys000
Thomass-MacBook-Pro:~ thomasbailey$ sudo kextstat | grep com.mice
Password:
Thomass-MacBook-Pro:~ thomasbailey$

**Comment by FranticRain** [comment]
Can you try `sudo kextload /Library/Extensions/360Controller.kext` and tell me what the output is?

**Comment by FranticRain** [comment]
This issue is being closed due to inactivity. We can't fix issues if we don't have enough information. So periodically we close down issues that can't be resolved if they seem to have been abandoned by their creators. If your issue still persists, please open a new issue after thoroughly consulting the README and providing all information as required by the README.
